app/views/listings/form/_currency_selector.haml

```diff
@@ -0,0 +1,2 @@
+.currency-selector
+  = Money.new(1, @current_community.currency).symbol
```

Comment:
This used to be a dropdown since in theory marketplaces supported multiple currencies. That feature was removed, so calling this `currency-selector` seems misleading.